src/app/Job-Service/job.service.spec.ts

```diff
@@ -1,12 +1,60 @@
 import { TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { JobService } from './job.service';
+import { of } from 'rxjs/internal/observable/of';
 
 describe('JobService', () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+  let jobService: JobService; // added for testing
+
+  beforeEach(() =>{
+    TestBed.configureTestingModule({
+    imports: [HttpClientTestingModule],
+    providers: [JobService] 
+  });
+  jobService = TestBed.get(JobService); //added for testing
+});
 
   it('should be created', () => {
-    const service: JobService = TestBed.get(JobService);
-    expect(service).toBeTruthy();
+    // const service: JobService = TestBed.get(JobService);
+    expect(jobService).toBeTruthy();
+  });
+  describe('getAllJobs', () => {
+    it('it should return all jobs', () => {
+      const jobResponse =[
+        {id: 1,
+        title: "Developer",
+        description: "test",
+        location: "Chicago",
+        isFilled: true,
+        skills: []
+        }];
+      let response;
+      spyOn(jobService, 'getAllJobs').and.returnValue(of(jobResponse));
+
+      jobService.getAllJobs().subscribe(res => {
```

Comment:
Is this test actually doing anything? It looks like you're spying getAllJobs first, and telling it to automatically return jobResponse. And then you're expecting that the forced response matches jobResponse? This seems like you're effectively testing that your mock is returning what you told it to return.